paper-input-container.html

```diff
@@ -260,18 +264,19 @@
         -webkit-appearance: none;
         text-align: inherit;
 
-        @apply(--paper-font-subhead);
```

Comment:
See answer in https://github.com/PolymerElements/paper-dialog-behavior/pull/61/files#r49791795